File: GameEngine_IOS/GameEngine/Commons/Vector2f.m
#import "Vector2f.h"

@implementation Vector2f


- (id)init : (float) aX : (float) aY
{
    self = [super init];
    if (self) {
        [self setX: aX];
        [self setY: aY];
    }
    return self;
}

@end

File: GameEngine_IOS/GameEngine/Commons/Vector2f.swift
import Foundation

/**
* Represents a 2D Vector with their x,y,z coordinates
*/
public class Vector2f {
    
    var x : Float;
    var y : Float;
    
    /**
    * The initialize of the vector
    *
    * @param x
    *            coordinate
    * @param y
    *            coordinate
    */
    public init(x : Float, y : Float) {
        self.x = x;
        self.y = y;
    }
}